server/plugins/renderer/responses/raw-response.js

```diff
@@ -14,24 +14,24 @@ const internals = {
  */
 function RawResponse(data, headers, statusCode) {
 
-    this.respond = function (request, reply) {
-        var payload = data;
+    this.respond = function (request, h) {
```

Comment:
what does h stand for ?

Reply:
@junedkazi [From the docs](https://hapi.dev/api?v=20.0.0#response-toolkit) "h" stand for HAPI.
They replaced the _reply_ function with this "Response Toolkit" (h) in v17.